tempesta_fw/str.h

```diff
@@ -74,7 +74,7 @@
 #define TFW_STR_CN_SHIFT	TFW_STR_FBITS
 #define __TFW_STR_CN_MAX	(~TFW_STR_FMASK)
 /* Str is compound from many chunks, use indirect table for the chunks. */
-#define __TFW_STR_COMPOUND 	(~((1U << TFW_STR_FBITS) - 1))
+#define __TFW_STR_COMPOUND	16
```

Comment:
First, the value is wrong. But even if it were the correct value, it should be defined the way it was before. In fact, the value is actually `(~(TFW_STR_FMASK))`.

Secondly, though, I don't think this constant is needed anymore. Please see a comment below.